lib/embed/media_tag.rb

```diff
@@ -127,7 +128,7 @@ def access_restricted_overlay(stanford_only, location_restricted)
       <<-HTML.strip_heredoc
         #{location_only_overlay(stanford_only, location_restricted)}
         <div class='sul-embed-media-access-restricted-container' data-access-restricted-message>
-          <div class='sul-embed-media-access-restricted'>
+          <div class='sul-embed-media-access-restricted' id='access-restricted-message-div'>
```

Comment:
You should be able to use `file_index` here to (and above in the `aria-labelledby` attribute) to give a unique id to these elements.